src/common/env.ts

```diff
@@ -4,3 +4,7 @@ export const isMac = process.platform === 'darwin';
 export const isM1 = isMac && (os.cpus()[0]?.model.includes('Apple M1') ?? false);
 
 export const isRenderer = typeof process !== 'undefined' && process.type === 'renderer';
+
+const env = { ...process.env };
+delete env.PYTHONHOME;
+export const sanitizedEnv = env;
```

Comment:
Does this actually work? I though envs were inherited System > User > Program, so doing deleting it from the program envs wouldn't do anything.

Reply:
It only matters for the spawned process, which you can control the env vars for. 

https://nodejs.org/api/child_process.html#:~:text=the%20child%20process.-,env%20%3CObject%3E%20Environment%20key%2Dvalue%20pairs.%20Default%3A%20process.env.,-argv0%20%3Cstring%3E